src/mpi.c

```diff
@@ -504,6 +504,13 @@ static void *load_library(const char *const libname) {
     assert(0);
   }
 
+#elif __FreeBSD__
+
+  // dlmopen is not supported, always use dlopen instead
```

Comment:
I think `dlmopen` is a glibc-only thing?

Reply:
Yes, it doesn't exist on FreeBSD. There's a run-time option to choose between `dlopen` and `dlmopen`, and I'm ignoring this on FreeBSD.

Reply:
What I wanted to suggest is to allow dlmopen only on glibc, rather than excluding on all platforms that don't support it, since it's more of an exception rather than the norm.

Reply:
I'm not excluding any platforms. I'm ignoring the option so that FreeBSD unconditionally uses `dlopen`. The alternative would be to emit an error on FreeBSD, but that doesn't seem worth the trouble.